scripts/epicsArchChecker

```diff
@@ -8,13 +8,29 @@ of reports:
 2. PVs with no alias and Aliases with no PVS.
 3. PVs no connected.
 4. Files that do not exist.
+
+The script also runs epicsArchVerify for LCLS2 hutches.
 """
 import argparse
 import os
+import subprocess
 
 import ophyd.signal
 from prettytable import PrettyTable
 
+LCLS2_HUTCHES = ["rix", "tmo", "ued", "txi"]
+LCLS1_HUTCHES = ["xpp", "xcs", "mfx", "cxi", "mec"]
+
+
+def is_lcls2(path):
+    path = str.lower(path)
+    if any(hutch in path for hutch in LCLS2_HUTCHES):
```

Comment:
I'm hoping that this never happens, but this could catch a path like: `/reg/g/tmo/abadxppfilepath/EpicsArch.txt`.  The check is brittle to begin with, but it's not unlike what we do elsewhere.  Maybe we could look for a hutch folder?

```python
if any(f"/{hutch}/" in path for hutch in LCLS2_HUTCHES):
```

or more robustly
```python
from pathlib import path

path_obj = Path(path)
if any(hutch in path_obj.parts for hutch in LCLS2_HUTCHES):
```